### DISCUSSION TITLE
Manage access of collaborators and provide for them different privileges in Discussion Panel

### DISCUSSION THREAD

**Discussion by kkochanski** [community/community#55620] [New to GitHub]
My goal is to have a private repository, where there's also a discussion panel. I would love to provide different privileges for every invited collaborator (like some edit all of posts, some can only their, some can remove all etc.). Is it feasible?

I'm creating some PoC, though I'm stuck at point of managing collaborators permissions. While basing on document - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository#changing-permissions-for-a-team-or-person - I should be able to do that, though at my panel, I can only add new collaborators or remove them. 

Appreciate your help folks!

![CleanShot 2023-05-16 at 11 23 44](https://github.com/community/community/assets/8296936/ca8d43fa-ea17-499c-bc8a-d6c346bee46c)

**Comment by queenofcorgis** [comment]
Hi there, great questions @kkochanski ⭐ 

Custom repository roles are only available to Enterprise users, do you have an Enterprise plan?

**Reply by exonomyapp** [reply]
What about allowing a team from an enterprise to access my private repo? Is that possible without enterprise?

**Comment by kkochanski** [comment]
Hello @queenofcorgis, indeed, I do not have an Enterprise plan. That makes sense, thank you for help! 💪
